tgui/packages/tgui/interfaces/GPS.tsx

```diff
@@ -0,0 +1,214 @@
+                  >
+                    {trackable ? 'Yes' : 'No'}
+                  </Button.Checkbox>
+                </LabeledList.Item>
+                <LabeledList.Item label="Send distress">
+                  <Button.Checkbox
+                    onClick={() => act('toggle_distress')}
+                    selected={distress}
+                    checked={distress}
+                  >
+                    {distress ? 'Yes' : 'No'}
+                  </Button.Checkbox>
+                </LabeledList.Item>
+                <LabeledList.Item label="Tracking">
+                  <Button onClick={() => act('track_gps')}>
+                    {tracking ? tracking : 'None'}
+                  </Button>
+                </LabeledList.Item>
+              </LabeledList>
+            </Stack.Item>
+          </Stack>
+        </Section>
+        <Section title="Tracking">
+          <Collapsible title="GPS Devices">
+            {!!gps_info.length && (
+              <Stack vertical>
+                <TrackableList gps_info={gps_info} />
+              </Stack>
+            )}
+          </Collapsible>
+          <Collapsible title="Implants">
+            {!!imp_info.length && (
+              <Stack vertical>
+                <TrackableList gps_info={imp_info} />
+              </Stack>
+            )}
+          </Collapsible>
+          <Collapsible title="Warp Beacons">
+            {!!warp_info.length && (
+              <Stack vertical>
+                <TrackableList gps_info={warp_info} />
+              </Stack>
+            )}
+          </Collapsible>
+        </Section>
+      </Window.Content>
+    </Window>
+  );
+};
+
+let distress_red = false;
+const TrackableList = (props) => {
+  const { act } = useBackend();
+  const { gps_info } = props;
+  const nodes: JSX.Element[] = [];
+
+  distress_red = !distress_red;
+
+  for (let i = 0; i < gps_info.length; i++) {
+    const node = (
+      <Box key={i} fontSize={0.8}>
+        <Stack.Item>
+          <Stack>
+            <Stack.Item grow>
+              <Stack vertical>
+                <Stack.Item>
+                  <strong>{gps_info[i].name}</strong>
+                </Stack.Item>
+                <Stack.Item>
+                  <em>{`Located at (${gps_info[i].x}), (${gps_info[i].y}) | ${gps_info[i].z_info}`}</em>
+                </Stack.Item>
+                {gps_info[i].distress !== null && !!gps_info[i].distress && (
+                  <Stack.Item textColor={distress_red ? 'red' : 'default'}>
+                    Distress Alert
+                  </Stack.Item>
+                )}
+              </Stack>
+            </Stack.Item>
+            <Stack.Item align="center">
+              <Button
+                onClick={() => act('track_gps', { gps_ref: gps_info[i].ref })}
+                fontSize={1}
+              >
+                Track
+              </Button>
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
+        <Divider />
```

Comment:
```suggestion
        <Stack.Divider />
```
Resolves to the same thing, currently, but should use `Stack`-specific one in case that's not true in future.

Reply:
Idk, it stops showing up with this change